stock_picking_assign_limit_days/models/res_config_settings.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 PlanetaTIC - Marc Poch <mpoch@planetatic.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
You have mixed licenses beween LGPL and AGPL, please choose one across the whole module.